docs/docs/workflows/steps/code/state/README.md

```diff
@@ -39,12 +39,16 @@ $checkpoint = "1";
 You can read data previously saved in `$checkpoint` like so:
 
 ```javascript
-if ($checkpoint) {
-  console.log($checkpoint);
-}
+const checkpoint = $checkpoint ?? "default value"
 ```
 
-### Example workflow - use `$checkpoint` to increment a number
+### Workflow state and concurrency
```

Comment:
Added this section to clarify how workflow state should be used with concurrency controls.